src/main/java/com/gregtechceu/gtceu/data/loader/forge/BedrockOreLoaderImpl.java

```diff
@@ -0,0 +1,12 @@
+package com.gregtechceu.gtceu.data.loader.forge;
+
+import com.gregtechceu.gtceu.data.loader.BedrockOreLoader;
+import com.gregtechceu.gtceu.data.loader.FluidVeinLoader;
+
+public class BedrockOreLoaderImpl extends BedrockOreLoader {
+
+    public BedrockOreLoaderImpl() {
+        super();
+        BedrockOreLoader.INSTANCE = this;
+    }
+}
```

Comment:
Is this class actually needed? Might just want to make `BedrockOreLoader` a singleton otherwise.